draft-ietf-quic-http.md

```diff
@@ -1589,15 +1593,9 @@ The entries in the following table are registered by this document.
 | NUM_PLACEHOLDERS             |  0x8   | {{settings-parameters}}   |
 | ---------------------------- | ------ | ------------------------- |
```

Comment:
Not a new issue but: RFC 7540 has a column for `Initial Value`. I think it makes sense that HTTP/3 does similar.

Reply:
Opened #2442.